.github/pr-custom-review.yml

```
@@ -19,11 +19,20 @@ rules:
     check_type: changed_files
     condition:
       include: .*
-      exclude: ^runtime/(kusama|polkadot)/src/[^/]+\.rs$
+      # excluding files from 'Runtime files' and 'CI team' rules
+      exclude: ^runtime/(kusama|polkadot)/src/[^/]+\.rs$|\.gitlab-ci.yml|scripts/ci/.*|\.github/.*
```

Missing `^` anchors and `\` to escape `.`

```suggestion
      exclude: ^runtime/(kusama|polkadot)/src/[^/]+\.rs$|^\.gitlab-ci\.yml|^scripts/ci/.*|^\.github/.*
```

---

Thank you